### PR TITLE
Add doc for event page

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-events.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+For each event, the following information is logged.
+- Event Type
+- Status
+- Message
+- Time
+- Triggered By
+
+> **Note:**
+>
+> - If the cluster events happened before the event feature added, these event are not visible on Event page.
```
Comment (ran-huang) [inline tidb-cloud/monitor-events.md:49]:
```suggestion
> - If the cluster events happened before 2023-03-01, these events are not visible on the Events page. 
```

File: tidb-cloud/monitor-events.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+> **Note:**
+>
+> - If the cluster events happened before the event feature added, these event are not visible on Event page. 
+> - TiDB Cloud will keep adding more event types in future release. 
+
+## Events retention policy
+
+- For Dedicated Tier clusters, the event data is kept for 7 days.
+- For Serverless Tier clusters, the event data is not availabe .
```
Comment (ran-huang) [inline tidb-cloud/monitor-events.md:55]:
```suggestion
- For Serverless Tier clusters, the event data is not available.
```

**Review by kaaaaaaang — APPROVED** [review]

File: tidb-cloud/monitor-events.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    > If you have multiple projects, you can switch to the target project from the ☰ hover menu in the upper-left corner.
+
+2. Click the name of the target cluster. The cluster overview page is displayed.
+3. Click **Events** in the left navigation pane.
+
+## Logged events
+
+TiDB Cloud logs the following types of cluster events:
+
+| Event Type| Description |  Added since |
```
Comment (ran-huang) [inline tidb-cloud/monitor-events.md:33]:
```suggestion
| Event Type| Description |  Available since |
```

File: tidb-cloud/monitor-events.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+|:--- |:--- |:--- |
+| Creating Cluster / Cluster Created |  Events recoded for cluster creation action.  |  2023-03-01   |
+| Pausing Cluster  / Cluster Paused  |   Events recoded for cluster pause action. |  2023-03-01   |
+| Resuming Cluster / Cluster Resumed  |   Events recoded for cluster resume action. |  2023-03-01   |
+| Modifying Cluster Size  / Cluster Size Modified  |   Events recoded for modify cluster size action. |  2023-03-01   |
```
Comment (ran-huang) [inline tidb-cloud/monitor-events.md:38]:
```suggestion
| Modifying Cluster Size  / Cluster Size Modified  |   Events recoded for modifying cluster size action. |  2023-03-01   |
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: tidb-cloud/tidb-cloud-events.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+# TiDB Cloud Events
+
+TiDB Cloud logs the historical events at the cluster level. An *event* indicates a change in the TiDB Cloud environment. You can review the logged events on the **Events** page, including the event type, status, message, trigger time, and trigger user.
+
+This document describes how to review the events for TiDB Cloud clusters using the **Events** page. 
+
+> **Note:**
+>
+> The Event page currently is only available for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-events.md:14]: Can you add this limitation to this doc: [Serverless Tier Limitations](https://docs.pingcap.com/tidbcloud/serverless-tier-limitations)

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/tidb-cloud-events.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+- Triggered By
+
+> **Note:**
+>
+> - If the cluster events happened before 2023-03-22, these events are not visible on the Events page. 
+
+## Event retention policy
+
+- For Dedicated Tier clusters, the event data is kept for 7 days.
+- For Serverless Tier clusters, the event data is not available.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:55]:
```suggestion
For Dedicated Tier clusters, the event data is kept for 7 days.
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-events.md:55]: As this feature is not provided for Serverless Tier clusters for now, there is no need to provide the retention info for Serverless Tier.

File: tidb-cloud/tidb-cloud-events.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+| CreateCluster |  Events recoded for cluster creation action.  |  2023-03-22   |
+| PauseCluster |   Events recoded for cluster pause action. |  2023-03-22   |
+| ResumeCluster |   Events recoded for cluster resume action. |  2023-03-22   |
+| ModifyClusterSize |   Events recoded for modifying cluster size action. |  2023-03-22   |
+
+For each event, the following information is logged:
+
+- Event Type
+- Status
+- Message
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:44]: What kind of info will be provided in Message?

File: tidb-cloud/tidb-cloud-events.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- Time
+- Triggered By
+
+> **Note:**
+>
+> - If the cluster events happened before 2023-03-22, these events are not visible on the Events page.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:50]:
```suggestion
> Cluster events started before 2023-03-22 are not visible on the Events page. 
```

File: tidb-cloud/tidb-cloud-events.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
+---
+title: TiDB Cloud Cluster Events
+summary: Learn how to review the events for TiDB Cloud clusters using the Event page. 
+---
+
+# TiDB Cloud Events
+
+TiDB Cloud logs the historical events at the cluster level. An *event* indicates a change in the TiDB Cloud cluster environment. You can review the logged events on the **Events** page, including the event type, status, message, trigger time, and trigger user.
+
+This document describes how to review the events for TiDB Cloud clusters using the **Events** page.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:10]:
```suggestion
This document describes how to view the events for TiDB Cloud clusters using the **Events** page and lists the supported event types.
```

File: tidb-cloud/tidb-cloud-events.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+---
+title: TiDB Cloud Cluster Events
+summary: Learn how to review the events for TiDB Cloud clusters using the Event page. 
+---
+
+# TiDB Cloud Events
+
+TiDB Cloud logs the historical events at the cluster level. An *event* indicates a change in the TiDB Cloud cluster environment. You can review the logged events on the **Events** page, including the event type, status, message, trigger time, and trigger user.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:8]:
```suggestion
TiDB Cloud logs the historical events at the cluster level. An *event* indicates a change in the TiDB Cloud cluster. You can view the logged events on the **Events** page, including the event type, status, message, trigger time, and trigger user.
```

File: tidb-cloud/tidb-cloud-events.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
+---
+title: TiDB Cloud Cluster Events
+summary: Learn how to review the events for TiDB Cloud clusters using the Event page. 
+---
+
+# TiDB Cloud Events
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:6]:
```suggestion
# TiDB Cloud Cluster Events
```

File: tidb-cloud/tidb-cloud-events.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
+---
+title: TiDB Cloud Cluster Events
+summary: Learn how to review the events for TiDB Cloud clusters using the Event page.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:3]:
```suggestion
summary: Learn how to view the events for TiDB Cloud clusters using the Event page. 
```

File: tidb-cloud/tidb-cloud-events.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+## Logged events
+
+TiDB Cloud logs the following types of cluster events:
+
+| Event Type| Description |  Available since |
+|:--- |:--- |:--- |
+| CreateCluster |  Events recoded for cluster creation action.  |  2023-03-22   |
+| PauseCluster |   Events recoded for cluster pause action. |  2023-03-22   |
+| ResumeCluster |   Events recoded for cluster resume action. |  2023-03-22   |
+| ModifyClusterSize |   Events recoded for modifying cluster size action. |  2023-03-22   |
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:38]:
```suggestion
| CreateCluster |  Events recoded for cluster creation actions.  |  2023-03-22   |
| PauseCluster |   Events recoded for cluster pause actions. |  2023-03-22   |
| ResumeCluster |   Events recoded for cluster resume actions. |  2023-03-22   |
| ModifyClusterSize |   Events recoded for modifying cluster size actions. |  2023-03-22   |
```

File: tidb-cloud/tidb-cloud-events.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+## Logged events
+
+TiDB Cloud logs the following types of cluster events:
+
+| Event Type| Description |  Available since |
+|:--- |:--- |:--- |
+| CreateCluster |  Events recoded for cluster creation actions.  |  2023-03-22   |
+| PauseCluster |   Events recoded for cluster pause actions. |  2023-03-22   |
+| ResumeCluster |   Events recoded for cluster resume actions. |  2023-03-22   |
+| ModifyClusterSize |   Events recoded for modifying cluster size actions. |  2023-03-22   |
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:38]:
```suggestion
| CreateCluster |  Events recorded for cluster creation actions.  |  2023-03-22   |
| PauseCluster |   Events recorded for cluster pause actions. |  2023-03-22   |
| ResumeCluster |   Events recorded for cluster resume actions. |  2023-03-22   |
| ModifyClusterSize |   Events recorded for modifying cluster size actions. |  2023-03-22   |
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**